app/views/topics/propose_archive.html.erb

```diff
@@ -1,10 +1,24 @@
 <%= tag_header @archival.tag, 'Archive topic' %>
```

Comment:
In the PR description you say that the content designers said: "'Delete' rather than 'Archive' because we won't keep a copy of the topic page".  Should this (and other visible buttons, etc) be updated to say Delete, therefore?